internal/runbits/buildscript/file.go

```diff
@@ -57,42 +55,48 @@ func Initialize(path string, auth *authentication.Auth) error {
 	if err != nil {
 		return errs.Wrap(err, "Unable to get the local commit ID")
 	}
+
 	buildplanner := buildplanner.NewBuildPlannerModel(auth)
-	expr, atTime, err := buildplanner.GetBuildExpressionAndTime(commitId.String())
+	script, err = buildplanner.GetBuildScript(commitId.String())
 	if err != nil {
 		return errs.Wrap(err, "Unable to get the remote build expression and time")
 	}
-	script, err = NewFromBuildExpression(atTime, expr)
+
+	sb, err := script.Marshal()
```

Comment:
I've renamed it to `scriptBytes`. Note I agree with your point of view though I'd caveat that it's ok as long as the var is immediately consumed right after, because then the full context is in view.